recipes/configs/llama4/scout_17B_16E_lora.yaml

```diff
@@ -0,0 +1,91 @@
+# Config for multi-device full finetuning in full_finetune_distributed.py
+# using a Llama4 17Bx16E MoE model
+#
+# This config assumes that you've run the following command before launching:
+#   tune download meta-llama/Llama-4-Scout-17B-16E-Instruct
+#
+# To launch on 8 devices, run the following command from root:
+#   tune run --nproc_per_node 8 lora_finetune_distributed --config llama4/scout_17B_16E_lora
+#
+# You can add specific overrides through the command line. For example, to use a larger bsz:
+#   tune run --nproc_per_node 8 lora_finetune_distributed --config llama4/scout_17B_16E_lora batch_size=8
+#
+# This config was only tested on 8xA100 machine.
+
+output_dir: /tmp/torchtune/llama4_17Bx16E/lora
+
+# Modeling Arguments
+model:
+  _component_: torchtune.models.llama4.lora_llama4_scout_17b_16e
+  lora_attn_modules: ['q_proj', 'v_proj', 'output_proj']
+  apply_lora_to_mlp: True
```

Comment:
Can you expose [these](https://github.com/pytorch/torchtune/blob/8a10fad5bf67adffe0c0d3a391b240a532116b07/recipes/configs/llama3_2_vision/11B_lora_single_device.yaml#L21) options here?